### DISCUSSION TITLE
How to reset updates for particular dependency

### DISCUSSION THREAD

**Discussion by robg-temp** [renovatebot/renovate#9583] [Request Help]
HI guys,

I have been testing out our configuration on a couple of specific dependencies.  Some renovate PRs I merged in, others, I deleted due to errors.  It seems that renovate has stopped updating a particular dependency.  I have incremented the version several times, yet, no PRs have been created for it in a while.  

**Is there a way to "reset" renovate for this one dependency OR in general?**

We are using renovate as a self-hosted Docker container that runs a central config file for about 10 projects.

Thanks!

Rob

**Comment by rarkins** [comment]
I suggest enabling `dependencyDashboard` so that you can see a list of pending/open/ignored PRs. If any are ignored then you can tick the corresponding checkbox to have them recreated the next run.

**Reply by robg-temp** [reply]
I'll give that a try right now.  Thanks!

**Reply by robg-temp** [reply]
I added the dependencyDashboard attribute (see below).  Where would I find the issue?  I did a search and couldn't find it:

![image](https://user-images.githubusercontent.com/80468671/115050546-daa77000-9ea9-11eb-8984-629d13492bbe.png)

```
module.exports = {
  "baseBranches": (process.env.ACTIVE_BRANCHES + ",master").split(","),
  "branchPrefix": "renovate/",
  "onboarding": true,
  "onboardingConfig": {
    "extends": [ "config:base" ]
  },
  "platform": "github",
  "rangeStrategy": "bump",
  "dependencyDashboard": true,
  "packageRules": [
```

Thanks!

**Reply by rarkins** [reply]
Issues will be created in each repo

**Reply by robg-temp** [reply]
Strange, there are no Issues anywhere.

**Reply by viceice** [reply]
Your token needs issue creation permission.

Check the container debug logs (stdout).
